examples/emberjs/todomvc/app/components/create.gjs

```diff
@@ -0,0 +1,32 @@
+import Component from '@glimmer/component';
+import { on } from '@ember/modifier';
+import { service } from '@ember/service';
+import { isBlank } from '@ember/utils';
+
+export default class Create extends Component {
+  <template>
+    <input
+      class="new-todo"
+      {{on 'keydown' this.createTodo}}
+      aria-label="What needs to be done?"
+      placeholder="What needs to be done?"
+      autofocus
+    >
+  </template>
+
+  @service repo;
+
+  // TODO: we should use a <form> instead of this.
```

Comment:
+1, this would be a larger refactor though, since the template as well as the styles package would need to change.

Reply:
yes, but it would be a correct change

Reply:
absolutely!
@FadySamirSadek - I know we talked about a11y changes (globally). 
Is there already an issue filed? If not, I can file one in the next few days ..